blog/_posts/2017-09-08-team_code_hoppers_blog.md

```diff
@@ -0,0 +1,51 @@
+
+#### Our accomplishments so far?
+
+In a nutshell, we learnt about coding, teamwork, collaboration and issue resolution. We even got to learn how to effectively communicate with remote teams across the world, listening and focusing on a common goal, defining tasks and keeping schedules. Well, sometimes not everything went as we had planned and from that we learnt that nothing is exactly perfect and we always have to keep striving to efficiently communicate and resolve our blockers. **It has been an amazing journey!**
+
+#### Our take from the summer!
+
+- We learned about [Rails](http://rubyonrails.org/) and [Elastic search](http://rubyonrails.org/)
+- Keeping and maintaining coding standards specified by an organisation
+- Learnt the importance of implementing peer to peer reviews
+- We felt the need of contributing to a shared vision
+- We revamped the search functionality by designing the UI and backend
+- We added new features to the new landing page - Cynthia helped alot with this
+- Simon helped us implement the backend [searchKick](https://github.com/ankane/searchkick) functionality
+
+![Cynthia's take!](/img/blog/2017/team_code_hoppers_cynthia.jpg)
+<div align="center" class="image-credits">Rgsoc has been a great experience. I have learnt so much. It has been fun collaborating with my partner, the coaches and our mentor. I hope to continue contributing to open farm (Photo by: Cynthia)</div>
+
+
+![Roselyne's take!](/img/blog/2017/team_code_hoppers_roselyne.jpg)
+<div align="center" class="image-credits">This was a fruitful summer and I would love to do it again. The lessons I have carried from this summer will go a long way in accelerating my growth as a software programmer. I have enjoyed the momentum that this summer has brought. (Photo by: Roselyne)</div>
+
+
+#### We wish to thank…
+
+- Our supervisor, [Simon](https://twitter.com/simonv3) - We got to work with Simon through out the OpenFarm project and we cannot thank him enough for the immense work experience we have gained from him. 
+- OpenFarm designer, [Sophia] - We got to work with Sophia on most designs that were implemented on the OpenFarm project. It felt great to learn about design from a person with a keen eye for design. Thanks Sophia, it was a pleasure to learn from you. 
+- [Travis Foundation](https://twitter.com/travis_fnd) - For sponsoring our program for the entire summer. We remain forever grateful, and assure you that we really did feel the impact of the trust and belief you placed upon us. We plan to continue sharing this experience with interested parties. 
+- We also widh to thanks our coaches - Sigu, William and Emmanuel for the guidance and time they dedicated towards the summer. These great guys  ensured that we resolved some of the issues we encountered during the program.
+- We also wish to thank everyone at the [Rails Girls Summer of Code](https://twitter.com/RailsGirlsSoC) of Code community who made this summer be as smooth, pleasant and awesome. 
```

Comment:
Actually now that whole 'Rails Girls Summer of Code' is link redirect text, you would need to remove extra 'of code'. :)